Library/Homebrew/utils.rb

```diff
@@ -9,6 +9,20 @@
 
 class Tty
   class << self
+    def tick
+      # necessary for 1.8.7 unicode handling since many installs are on 1.8.7
+      @tick ||= ["2714".hex].pack("U*")
+    end
+
+    def cross
+      # necessary for 1.8.7 unicode handling since many installs are on 1.8.7
+      @cross ||= ["2718".hex].pack("U*")
+    end
```

Comment:
Just to be clear, I originally suggested to move the constants into the class. But I definitely prefer the look of `Tty.cross` over `Tty::UNICODE_CROSS`, even if it may come at a certain penalty (I'd guess it is marginally slower).